data/migrations/20190122212655_recipe_table.js

```diff
@@ -0,0 +1,13 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('recipes', table =>{
+      table.increments();
+      table.string('recipe_name').notNullable().unique();
+      table.integer('dish_id').unsigned();
+      table.foreign('dish_id').references('id').on('dishes');
```

Comment:
You seem to use this same setup for foreign keys elsewhere; why not make a helper function for it? For example:
```javascript
// In a new file, data/migrations/helpers.js
module.exports = {
  foreign: (table, name, ref) => {
    table.integer(name).unsigned();
    table.foreign(name).references('id').on(ref);
  }
};
```
and the usage, in this current file for example:
```javascript
const { foreign } = require('./helpers.js');

exports.up = knex => {
  return knex.schema.createTable('recipes', table => {
    table.increments();
    table.string('recipe_name').notNullable().unique();
    foreign(table, 'dish_id', 'dishes');
  });
};

exports.down = knex => knex.schema.dropTableIfExists('recipes');
```